test/models/repository_test.rb

```diff
@@ -268,12 +268,13 @@ def test_as_json
     end
 
     def test_units_for_removal_yum
-      rpms = @fedora_17_x86_64.rpms.sample(2).sort
-
+      rpms = @fedora_17_x86_64.rpms.sample(2)
+      rpm_ids = rpms.map(&:id).sort
+      rpm_uuids = rpms.map(&:uuid).sort
```

Comment:
Can we preserve the previous empty line that separated the test data from the assertions?